meta-mender-client-only/recipes-mender/mender-client/mender-client_%.bbappend

```diff
@@ -0,0 +1,16 @@
+do_configure:prepend() {
+	if [ -n "${MENDER_FEATURES}" ]; then
+		bbwarn "having both the Mender integration and the meta-mender-client-only layer enabled will cause unexpected effects!"
+	fi
+}
+
+do_compile:append() {
+    echo "device_type=${MACHINE}" > ${B}/device_type
```

Comment:
This should use `MENDER_DEVICE_TYPE`.

Reply:
´MENDER_DEVICE_TYPE` is not set without the `INHERIT += "mender-xxx"` addition, see other comment.

Reply:
Hmm, ok. Maybe we can consider changing this though. I don't see any harm in setting a variable, even if no features are enabled. And it would enable this to work in the expected way.